packages/terra-form-select/src/single/Menu.jsx

```diff
@@ -260,7 +253,13 @@ class Menu extends React.Component {
     }
 
     if (select) {
-      select.focus();
+      if (!SharedUtil.isSafari()) {
+        this.props.select.focus();
+      } else {
+        setTimeout(() => {
+          this.props.select.focus();
```

Comment:
```suggestion
     select.focus();
```
Since select is already extracted from props on line no. 249